lib/generators/suspenders/inline_svg_generator.rb

```diff
@@ -0,0 +1,18 @@
+module Suspenders
+  module Generators
+    class InlineSvgGenerator < Rails::Generators::Base
+      include Suspenders::Generators::APIAppUnsupported
+      source_root File.expand_path("../../templates/inline_svg", __FILE__)
+      desc "Installs inline_svg"
```

Comment:
I should have mentioned this in the contributing section of the pull request description, but the current release has descriptions. We might want to use the [existing description](https://github.com/thoughtbot/suspenders/blob/main/templates/descriptions/inline_svg.md) for consistency's sake.

Reply:
Thanks! I'll change it here and in the documentation.